skopt/plots.py

```diff
@@ -368,7 +397,12 @@ def plot_evaluations(result, bins=20):
     for i in range(space.n_dims):
         for j in range(space.n_dims):
             if i == j:
-                ax[i, i].hist(samples[:, j], bins=bins,
+                if 'log-uniform' == space.dimensions[j].prior:
+                    b = space.bounds[j]
```

Comment:
Maybe -> `low, high = space.bounds[j]`

(Part of Tim's campaign for nice variable names everywhere :) )